MediaSession.cpp

```diff
@@ -560,7 +557,6 @@ CDMi_RESULT MediaKeySession::Decrypt(
       ChkArg(f_pbIV != NULL && f_cbIV == sizeof(DRM_UINT64));
 #else
       err = Drm_Reader_InitDecrypt(m_oDecryptContext, nullptr, 0);
-#endif
```

Comment:
Please do it differently and enable/disable whole blocks. If it's InitDecrypt() which is missing in PR 3_3 wrap whole if else with ifndef PR_3_3. The code in if (!initWithLast15) wrapper in PR_3_3 define isn't PR version specific a all

Reply:
Yes, code inside `if (!initWithLast15)` is argument validation which is not PR version specific. So making this applicable to all version and `InitDecrypt()` sequence is to be under `#ifndef PR_3_3`